packages/redwood/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@vercel/redwood",
-  "version": "0.6.0",
+  "version": "1.0.0-canary.17",
```

Please let me know if you'd like me to update this version to 1.0.0 or something your CI will handle

---

Let's leave it untouched for now. The release process bumps the version, but we'll be sure to do a major bump after we merge this.

```suggestion
  "version": "0.6.0",
```